### ISSUE TITLE
jfrog ds rbc does not error when non 200 http codes returned

### ISSUE THREAD

**Issue by ryancurrah** [jfrog/jfrog-cli#1263]
**Describe the bug**
When running the command `jfrog ds rbc` to create a new bundle it won't error out on non 200 responses from artifactory. 

**To Reproduce**

```
╰─ export JFROG_CLI_LOG_LEVEL=DEBUG
╰─ jfrog ds rbc --url="https://artifactory.acme.com/distribution" --user="notreal" --password="notreal" --passphrase="fake" --spec=aql.spec --sign my-bundle-name 1.0
.0
[Info] Creating: my-bundle-name/1.0.0
[Debug] Sending HTTP POST request to: https://artifactory.acme.com/distribution/api/v1/release_bundle
[Debug] Sending HTTP POST request to: https://artifactory.acme.com/distribution/api/v1/release_bundle

╰─ echo $?
0
```

**Expected behavior**

An error to be printed to stdout and an non-zero exit code.

**Versions**
- JFrog CLI version: 
```
╰─ jfrog --version
jfrog version 2.4.1
```

- JFrog CLI operating system:
```
╰─ uname -iop
i386 MacBookPro15,1 Darwin
```

- Artifactory Version:
```
7.25.X
```

**Additional context**

I'm guessing there is no code to check HTTP status code for the distribution code.

**Comment by yahavi** [comment]
@ryancurrah,
Thanks for reporting this issue!
#1271 was created to resolve this issue.
We'll keep you updated.

**Comment by ryancurrah** [comment]
Looks like fix has been released in version 2.5.1, closing.
